mesh/nurbs.hpp

```diff
@@ -65,6 +65,11 @@ class KnotVector
        order @a order and number of control points @a NCP. */
    KnotVector(int order, int NCP);
 
+   /** @brief Create a KnotVector by passing in a degree, a list of double interval
+       lengths of lenght n, and a list of continuity of length n + 1 */
+   KnotVector(int order, const Array<double> intervals,
```

Comment:
We usually use `mfem::Vector` instead of `Array<double>`.

Reply:
I've updated this to `mfem::Vector` (and modified the tests calling it accordingly).